lib/clio_audio_visualizer.rb

```diff
@@ -2,10 +2,9 @@
 
 require "clio_audio_visualizer/version"
 require "clio_audio_visualizer/metrics/metric"
-require "clio_audio_visualizer/metrics/file_reader"
-require "clio_audio_visualizer/metrics/datadog_reader"
+require "clio_audio_visualizer/metrics/file_stream"
+require "clio_audio_visualizer/metrics/datadog_stream"
```

Comment:
Should these still live in `Metrics::`? Should we create a `streams/` directory instead?